docs/guides/custom-redirects.mdx

```diff
@@ -37,12 +37,12 @@ If you would like to override the `redirect_url` value and supply a custom redir
 
 <CodeBlockTabs type="framework" options={["Next.js", "Remix"]}>
 ```sh filename=".env.local"
-NEXT_PUBLIC_CLERK_SIGN_IN_FORCE_REDIRECT_URL=/dashboard
+NEXT_PUBLIC_CLERK_SIGN_IN_FALLBACK_REDIRECT_URL=/dashboard
```

Comment:
This exists under the "Force environment variable redirects" section - so we should only the introduction of these. `SIGN_IN_FORCE_REDIRECT_URL` and `SIGN_UP_FORCE_REDIRECT_URL` are the force env variables.

If we want to incorporate Nikos's suggestion of providing a typical use case for these, I left a [suggestion on the original slack thread.](https://clerkinc.slack.com/archives/C01QFRQNHSN/p1714147838039329?thread_ts=1714083778.463479&cid=C01QFRQNHSN)

I think his suggestion belongs in the Onboarding guide.

```suggestion
NEXT_PUBLIC_CLERK_SIGN_IN_FORCE_REDIRECT_URL=/dashboard
```